web-services/services/auditing/src/main/resources/application.yml

```diff
@@ -15,7 +15,7 @@ spring:
   jpa:
     database-platform: org.hibernate.dialect.MySQL5InnoDBDialect
     hibernate:
-      ddl-auto: update
+      ddl-auto: create
```

Comment:
Guess this is responsible for wiping the database every time the environment is restarted?

Reply:
Yes we can change it.